spec/fixtures/sample_dump.sql

```diff
@@ -0,0 +1,176 @@
+--
```

Comment:
just a very basic dump file created for this purpose

may be desirable in the future to also test
- different pg dump versions
- more exotic column types
- may want some data that could intentionally try to break `line_regex`, a la bobby tables https://xkcd.com/327/

Reply:
Hi, I haven't looked at the test jet, but I started to use the gem (thanks for creating it), and I had an error (extra data after last expected column) that may be related to having commas inside text columns. 
Reading this made me think that a dumb file could be created and tested for this situation.
Also, I had to specify plain text format and utf-8 for my dumb file to avoid weird characters.

I guess I'll read more to confirm if this is a bug unless someone had experienced it before and reply with a bit of advice.